views/building-hours/building.js

```diff
@@ -16,75 +15,106 @@ import * as c from '../components/colors'
 import type {BuildingInfoType} from './types'
 import {isBuildingOpen} from './is-building-open'
 import {formatBuildingHours} from './format-building-hours'
+import CollapsibleBlock from '../components/collapsibleBlock'
+import {allBuildingHours} from './all-building-hours.js'
+import Dimensions from 'Dimensions'
```

Comment:
I think this also exists as an import from "react-native" itself; could you import from there, please?